zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/CollectionDetails.java

```diff
@@ -79,9 +80,15 @@ public CollectionDetail get(HttpServletRequest request, HttpServletResponse resp
 
         Set<Integer> teamIds = Root.zebedee.getPermissionsService().listViewerTeams(collection.description, session);
         List<Team> teams = Root.zebedee.getTeamsService().resolveTeams(teamIds);
-        teams.forEach(team -> {
-            collection.description.teams.add(team.getName());
-        });
+
+        // The members property will always be empty because we're using the Team class, rather than creating a new one
```

Comment:
Not sure I follow this. Doesn't teams ^^^ have populated values already? Why do we need to create a new list for the team details? Why can't we just use the teams?

Reply:
Fair question. The main reasons were (and I'm happy to be told I'm wrong on this):
1) We're not 100% sure what in old Florence is using the existing teams array and there's a high chance we'd unknowingly break something.
2) The teams property has a type of `List<String>`, which doesn't match our use (we need `List<Team>` at least) but it's in the base collection class, which I don't particularly want to edit because it's extended by quite a few other classes.